sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -217,8 +217,13 @@ public String toString() {
     }
   }
 
+  /**
+   * A Sampler that uses the sampled flag of the parent Span, if present. If the span has no parent,
+   * this Sampler will use the "root" sampler that it is built with. See documentation on the {@link
+   * Builder} methods for the details on the various configurable options.
+   */
   @Immutable
-  static class ParentBased implements Sampler {
+  public static class ParentBased implements Sampler {
```

Comment:
Do we need to make the class public?

Reply:
we do, if you want to use the builder, otherwise you can't import the inner class builder

Reply:
Ah got it

Reply:
(as an aside, Samplers.ParentBased.Builder : 🤮 )